CoolCurrency/Controller/SettingsViewController.swift

```diff
@@ -196,6 +200,27 @@ extension SettingsViewController {
         present(alert, animated: true, completion: nil)
     }
     
+    private func resetEmailAlert() {
+        var textField = UITextField()
+        let alert = UIAlertController(title: "Set Email",
```

Comment:
You can also put this in an extension to make things a little more streamlined. Junaid's last PR has a good example if you want to check it out.